tidy3d/components/medium.py

```diff
@@ -2775,6 +2788,12 @@ class CustomAnisotropicMedium(AbstractCustomMedium, AnisotropicMedium):
         description="This field is ignored. Please set ``allow_gain`` in each component",
     )
 
+    subpixel: bool = pd.Field(
```

Comment:
Just a suggestion if you agree: add a validator here that guarantees this value is set to `None`, so that the user is reminded that this field is ignored in case they forget and use it.

Reply:
Yeah, a warning should be fine. No reason to error out.